### ISSUE TITLE
vl-view center prop problem

### ISSUE THREAD

**Issue by fnoop** [ghettovoice/vuelayers#43]
I'm using vl-view component like this:
```
vl-map(data-projection="EPSG:4326" ref="map" :load-tiles-while-animating="true" :load-tiles-while-interacting="true", :controls="{attribution: false, zoom: false}")
    vl-view(v-if="xycenter.length > 0" :zoom="mapZoom" :center="xycenter" :rotation="0")
```
where `xycenter` is a computed property that contains [long, lat] in 4326 projection.

However it seems this computed property needs to be set several times before the vl-view displays the correct centered map.  It looks like for at least the first time, the vl-view computed properties are not correct:
```
resolvedDataProjection:"EPSG:4326"
viewCenter:Array[2]
0:0.0013400196419961843
1:-0.0003176205455872605
viewCenterViewProj:Array[2]
0:149.1703042
1:-35.3573574
viewProjection:"EPSG:3857"
viewResolution:4.777314267823516
viewRotation:0
viewZoom:15
```
This is when the center prop is set to:
```
center:Array[2]
0:149.1703042
1:-35.3573574
```
---

After being setting the xycenter computed property once or twice more, then the vl-view computed properties are correct and the map is centered as expected:
```
resolvedDataProjection:"EPSG:4326"
viewCenter:Array[2]
0:149.1703039
1:-35.35735649999999
viewCenterViewProj:Array[2]
0:16605562.27162587
1:-4212551.134875238
viewProjection:"EPSG:3857"
viewResolution:4.777314267823516
viewRotation:0
viewZoom:15
```
The center prop is now set to:
```
center:Array[2]
0:149.1703039
1:-35.3573565
```

**Comment by ghettovoice** [comment]
Something wrong with data projection resolving. I'll test this today once more.

**Comment by ghettovoice** [comment]
Fixed in v0.10.4, please upgrade.
Re-open if you still see incorrect behavoir
